.travis.yml

```diff
@@ -73,11 +129,15 @@ jobs:
       - sudo ./tools/travis/run_tests.sh tests/integration/snapd
     # CLA check, only in pull requests, not coming from the bot.
     - if: type = pull_request AND sender != snappy-m-o
-      script: ./tools/travis/run_cla_check.sh
+      script:
+        - sudo apt install -y git python-launchpadlib
+        - ./tools/cla_check.py
```

Comment:
Now that snaps can run on trusty you can probably use the cla-check snap, no?

Reply:
Sort of, we should add a task for that, but there a bit more things that go in there.